### PR TITLE
csvlint: Add csvlint engine

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
Adds missing csvlint using ruby image.

Closes https://github.com/repometric/linterhub-docker/issues/10

### CODE REVIEW

File: engines/csvlint/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM linterhub/image-ruby
+LABEL maintainer="linterhub@repometric.com"
+RUN apk add --no-cache libcurl && \
+    apk add --update make \
```
Comment (yukiisbored) [inline engines/csvlint/Dockerfile]: Why do you need update?

File: engines/csvlint/tests/test.csv [hunk 2]
```diff
@@ -0,0 +1,4 @@
+"Foo","Bsr","Baz"
+
+"Biff","Baff","Boff"
+"Qux","Teaspoon","Doge"
```
Comment (yukiisbored) [inline engines/csvlint/tests/test.csv:4]: Missing new line on EOF.

Comment (nalinbhardwaj) [reply engines/csvlint/tests/test.csv:4]: On purpose, it will emit one ~more~ less error if added...

File: engines/csvlint/Dockerfile (hunk 1 above)
Comment (nalinbhardwaj) [reply engines/csvlint/Dockerfile]: csvlint needs `make` , the build fails otherwise.

File: engines/csvlint/tests/test.csv (hunk 2 above)
Comment (yukiisbored) [reply engines/csvlint/tests/test.csv:4]: Ah sorry, forgot this was a test case :P

File: engines/csvlint/Dockerfile (hunk 1 above)
Comment (yukiisbored) [reply engines/csvlint/Dockerfile]: I doubt you need --update to get make.

Comment (nalinbhardwaj) [reply engines/csvlint/Dockerfile]: Yes, you are right, it worked without `--update`. Removing.

**Comment by yukiisbored** [comment]
gah, I can't access the Docker cloud build. @xferra

**Review by xferra — APPROVED** [review]
Thank you for your first contribution, great job! Looks good to me!

**Pull request merged**